File: src/main/java/org/evertones/persistence/modules/product/ProductSpecification.java
package org.evertones.persistence.modules.product;

import com.querydsl.core.types.dsl.BooleanExpression;
import org.evertones.model.modules.product.QProduct;
import org.evertones.model.types.ProductType;

public class ProductSpecification {

    static BooleanExpression queryProductTypeNotProduct() {
        return QProduct.product.productType.ne(ProductType.PRODUCT);
    }
}

File: src/main/scala/org/evertones/persistence/modules/product/ProductSpecification.scala
package org.evertones.persistence.modules.product

import org.evertones.model.modules.product.QProduct
import org.evertones.model.types.ProductType

object ProductSpecification {

    private[product] def queryProductTypeNotProduct = QProduct.product.productType.ne(ProductType.PRODUCT)

}
